spec/connect/client_spec.rb

```diff
@@ -1,75 +1,23 @@
 require 'spec_helper'
 
 describe SUSE::Connect::Client do
-  let :config do
-    SUSE::Connect::Config.new
-  end
-
-  subject { SUSE::Connect::Client.new(config) }
+  let(:config) { SUSE::Connect::Config.new }
+  let(:instance) { described_class.new config }
 
-  let(:default_logger) { SUSE::Connect::GlobalLogger.instance.log }
-  let(:string_logger) { ::Logger.new(StringIO.new) }
+  subject { instance }
 
   describe '.new' do
-    context :empty_opts do
-      it 'should set url to default_url' do
-        expect(subject.config.url).to eq SUSE::Connect::Config::DEFAULT_URL
-      end
-    end
-
-    context :passed_opts do
-      it 'should set insecure flag from options if it was passed via constructor' do
-        config.insecure = true
-        client = Client.new(config)
-        expect(client.config.insecure).to be true
-      end
-
-      it 'should set write_config flag from options if it was passed via constructor' do
-        config.write_config = true
-        client = Client.new(config)
-        expect(client.config.write_config).to be true
-      end
-
-      it 'allows to pass arbitrary options' do
-        config.foo = 'bar'
-        client = Client.new(config)
-        expect(client.config.foo).to eq 'bar'
-      end
-    end
-
-    context :from_config do
-      subject do
-        SUSE::Connect::Client.new(SUSE::Connect::Config.new)
-      end
-
-      before do
-        allow_any_instance_of(SUSE::Connect::Config).to receive(:read).and_return(
-          'url'      => 'https://localsmt.domain.local',
-          'language' => 'RU',
-          'insecure' => true
-        )
-      end
-
-      it 'should set url to the config URL' do
-        expect(subject.config.url).to eq 'https://localsmt.domain.local'
-      end
-
-      it 'should set token to one from config file' do
-        expect(subject.config.insecure).to eq true
-      end
-
-      it 'should set language to one from config file' do
-        expect(subject.config.language).to eq 'RU'
-      end
```

Comment:
this all just repeats the `SUSE::Connect::Config` tests and has actually nothing to do with `Client`